adobe/spectrum-web-components
# fix(color-handle,color-loupe,swatch,thumbnail): use the Opacity Checkerboard package

## Description
- leverage the new `opacity-checkerboard` class across various components
- also build an "is opacity checkerboard" build for use in Color Handle
- custom build Color Loupe so that it uses it as well
- Color Slider is skipped here for #3507 and lack of support for that functionality

## How has this been tested?
    1. Go [here](https://use-opacity-checkerboard--spectrum-web-components.netlify.app/storybook/?path=/story/color-handle--open)
    2. See it in action in Color Handle and Color Loupe
    1. Go [here](https://use-opacity-checkerboard--spectrum-web-components.netlify.app/storybook/?path=/story/thumbnail--landscape)
    2. See it in action in a Thumbnail
    1. Go [here](https://use-opacity-checkerboard--spectrum-web-components.netlify.app/storybook/?path=/story/swatch--opacity)
    2. See it in action in a Swatch

## Types of changes

## Checklist
